### PR TITLE
 Added feature to change Background

### LINKED ISSUE #420: To add a feature of custom background
**Actual Behaviour**

Only one background feature available

**Expected Behaviour**

To be able to add custom images in the background

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by stealthanthrax**
Fixes #420 

Changes: 
A feature to change the background was added

Surge Deployment Link: spotted-circle.surge.sh
Screenshots for the change:
screen shot 2018-05-09 at 11 26 44 pm

Next Steps:
Try to sync it with the server

### CODE REVIEW

**Comment by Akshat-Jain** [comment]
@stealthanthrax Please check the Travis CI build failure.

**Comment by stealthanthrax** [comment]
Done @Akshat-Jain

File: src/components/Settings/Settings.js [hunk 1]
```diff
@@ -105,6 +104,29 @@ class Settings extends React.Component {
             });
         }
     }
+    changeBack = () => {
+        let backgroundImage = document.getElementById('backgroundImgUrl').value;
+        console.log(backgroundImage);
```
Comment (anshumanv) [inline src/components/Settings/Settings.js]: Last one remains. :smile:

File: src/components/Settings/Settings.js [hunk 2]
```diff
@@ -105,6 +104,29 @@ class Settings extends React.Component {
...
+            if(cookies.get('loggedIn')===null||cookies.get('undefined')){
+                if(backgroundImage){
+                localStorage.setItem('anonymousImage',backgroundImage);
+                }
+                else{
+                    localStorage.setItem('anonymousImage','');
+                }
+            }
+            else {
+                /* eslint-disable */
```
Comment (anshumanv) [inline src/components/Settings/Settings.js]: What was the error that you disabled eslint? :thinking:

Comment (stealthanthrax) [reply src/components/Settings/Settings.js]: It was not considering the else statement and was giving the error of lonely if statement.

**Comment by stealthanthrax** [comment]
@anshumanv ,done 🤘

**Comment by stealthanthrax** [comment]
@dynamitechetan @madhavrathi @saurabhjn76 , can you guys please have a look?

**Comment by mariobehling** [comment]
Same comment as for theming applies here. Closing it. Please read more here: https://github.com/fossasia/susi_skill_cms/pull/448#issuecomment-390712677

**Pull request closed without merging**